deployments/deploy-ssh.sh

```diff
@@ -312,7 +312,7 @@ fi
 
 JUNIT_REPORT_PATH="/testresults/junit_basic.xml"
```

Comment:
I don't think for Openshift you can mount things to `/` dir, its gotta be something like `/tmp/` or `/mnt/`, please try this code on OCP jobs before merging

Reply:
Updated the description. Please review.